bigquery_etl/cli/query.py

```diff
@@ -2199,12 +2205,24 @@ def deploy(
         query_files = paths_matching_name_pattern(
             name, ctx.obj["TMP_DIR"], project_id, ["query.*", "script.sql"]
         )
-        if not query_files:
+        metadata_files = paths_matching_name_pattern(
+            name, ctx.obj["TMP_DIR"], project_id, ["metadata.yaml"]
+        )
+        if not query_files and not metadata_files:
```

Comment:
I like that metadata is the minimum required, so that only `schema.yaml` would make the check fail.